src/app/shared/warning-message/invalid-query/invalid-query.component.scss

```diff
@@ -0,0 +1,3 @@
+:host {
+  width: 100%;
```

Comment:
Styles that affect the surroundings of a component should be defined in the parent component.

If you define some padding in the parent element of this component, this `width` will break styles. Better approach would be to use `w-100` on this component's element in the parent component.